### ISSUE TITLE
Incorrect system/component Id on Evaluation and Submission Confirmation Reports for NONQADB TEE

### ISSUE THREAD

**Issue by esaber76** [US-EPA-CAMD/PI-1#6066]
The referenced System/Component Id field in the Evaluation and Submission Confirmation Reports emails are incorrect for a NONQADB TEE.

Submitted a new NONQADB TEE for ORIS 634, 4C for component Id GDB. The reports reference Id 421.

Evaluation Report:

<img width="623" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN1lDQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c62fa391864792b6046a0d38134d44c226061078/image.png" alt="image.png" />

Submission Confirmation Report:

<img width="622" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN2NDQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--dcbc33797caa1b54469b7e4f8a56944caf486d0b/image.png" alt="image.png" />

**Comment by annalbrecht** [comment]
Make sure check log is getting the right info, and then make sure the report is using that information. Also need to document how the reporting works.

**Comment by esaber76** [comment]
Evaluation report emails for TEE's are not currently sending in tst. Likely blocked by #5954.

Currently able to see the problem with submission confirmation emails.

Submitted TEE for component ID GDB:
![image](https://github.com/US-EPA-CAMD/easey-ui/assets/46533513/ca83a2e7-d45c-4494-b9e3-2dd4986be637)

Submission confirmation email referencing component ID 421 instead:
![image](https://github.com/US-EPA-CAMD/easey-ui/assets/46533513/a0aa1e2e-a60e-44f5-83e8-1ee5d4ba90ef)

**Comment by ergjustin** [comment]
it won't be possible to test it in DEV until the CDX bypass is removed, because the "to" email address is provided by CDX.

**Comment by esaber76** [comment]
@ergjustin - do you happen to know if the change for TEE would also include QA test and QA cert events? I happened to notice this issue for those test types yesterday.

![image](https://github.com/US-EPA-CAMD/easey-ui/assets/46533513/a4565936-d6e6-4f60-990d-c529f4c06805)

**Comment by ergjustin** [comment]
@esaber76 
Yes, this change will include TEE, QA test and QA cert events. This change was not deployed yet in test env

**Comment by esaber76** [comment]
Great, thanks!

**Comment by mxtomoto1** [comment]
Added new linearity test, new QA Cert Event (component only), and TEE (component only) for ORIS 634, unit 4C, 2023 Q3. Component IDs were correct in evaluation report email and submission confirmation email bodies.

Evaluation email:
<img width="759" alt="evaluation email" src="https://github.com/US-EPA-CAMD/easey-ui/assets/94564355/727a1a0c-2bcd-4f70-bdb5-30d6c99139d4">

Submission email:
<img width="743" alt="submission email" src="https://github.com/US-EPA-CAMD/easey-ui/assets/94564355/ac663b94-5ee0-4da9-a1ce-0d2e1e2b5820">

**Comment by ntknguyen** [comment]
I verified this issue is fixed.
